src/library/components/BoldMatch.tsx

```diff
@@ -21,12 +21,14 @@ const BoldMatch = (props: Props): JSX.Element => {
       {matchArray.map((match, i) => {
         if (match.toLowerCase() === props.match.toLowerCase()) {
           return (
-            <span key={'key' + match} className="bsds-font-weight-heavy">
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
This is tricky, but should be safe here since we know matchArray wont be re-ordered